explore/src/main/scala/explore/targeteditor/AladinToolbar.scala

```diff
@@ -27,7 +27,7 @@ import react.semanticui.sizes._
 final case class AladinToolbar(
   fov:                 Fov,
   current:             Coordinates,
-  loadingGSCandidates: Boolean,
+  loadingGSCandidates: Either[String, Boolean],
```

Comment:
Not a big deal, but this seems like it could be `Option[Boolean]`.

Reply:
I guess that's right. I'm not really showing the error msg